examples/app.py

```diff
@@ -32,7 +32,20 @@
    $ cd examples
    $ flask -a app.py db init
    $ flask -a app.py db create
+   $ flask -a app.py load_fixture
```

Comment:
Cosmetics: since we have `db init` and `db create`, using schema `<namespace> <verb>`, what about `fixtures load`?